be/src/exec/vectorized/aggregator.cpp

```diff
@@ -605,6 +595,41 @@ void Aggregator::_evaluate_agg_fn_exprs(vectorized::Chunk* chunk) {
     }
 }
 
+bool is_group_columns_fixed_size(std::vector<ExprContext*>& group_by_expr_ctxs,
+                                 std::vector<GroupByColumnTypes>& group_by_types, size_t* max_size, bool* has_null) {
+    size_t size = 0;
+    *has_null = false;
+
+    for (size_t i = 0; i < group_by_expr_ctxs.size(); i++) {
+        ExprContext* ctx = group_by_expr_ctxs[i];
+        if (group_by_types[i].is_nullable) {
+            *has_null = true;
+            size += 1; // 1 bytes for  null flag.
```

Comment:
why plus 1 is needed?

Reply:
because nullable column takes 1 ex tra byte to represent this value is null or not.